CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v0.11.0
+
+* Breaking Changes
+  * This release includes a change to how CA certificates are used in the connection
+    to NervesHub. If you are connecting to the publicly hosted https://nerves-hub.org,
+    then no changes are required.
+    If you are manually supplying `:ca_certs` config value to connect to another instance
+    of NervesHub, then you will need to update you config following the new instructions
+    for [Configuration](README.md#configuration) in the README
```

Comment:
I think this needs to be added to the README (it currently doesn't exist)

Reply:
Updated it to point to the NervesHubUserAPI docs